draft-ietf-rats-eat.md

```diff
@@ -952,6 +952,10 @@ to provide to the recipient cryptographically-verifiable proof that the sender h
 of a key.  This kind of attestation may be neceesary to verify the
 security state of the entity storing the private key used in a PoP application.
 
+6 -- Attesting Environment
+: This is used only in submodules. It indicates that the target of the 
+Attestation Evidence is itself an Attesting Environement. See {{layered}}.
```

Comment:
Not the current Attesting Environment that generated that Evidence, tough, but a "potential future" one in an upcoming "secure state transition" to the next layer. Maybe this could be read as "an Attesting Environment generates Evidence about itself", which it does not do?

Reply:
Regarding lines 955 - 957: The target environment contains an attesting environment but could contain other code, setup vectors, files or code that has functionality other than that used by an attesting environment. Structures representing a layer 'target environment' should allow for multiple disparate measurements.